index.js

```diff
@@ -0,0 +1,139 @@
+'use strict'
+
+const inherits = require('util').inherits
+const fp = require('fastify-plugin')
+const lru = require('tiny-lru')
+
+const OPEN = 'open'
+const HALFOPEN = 'half-open'
+const CLOSE = 'close'
+
+function circuitBreaker (fastify, opts, next) {
```

Comment:
Can you use a different function name here? It is already used for the decorator